Development/src/pages/nodes/NodesShow.js

```diff
@@ -0,0 +1,159 @@
+                        render={record =>
+                            Object.keys(record.tags).length > 0
+                                ? MapTags(record)
+                                : null
+                        }
+                    />
+                )}
+                <hr />
+                <UrlField source="href" label="Address" />
+                <TextField source="hostname" />
+                {controllerProps.record && QueryVersion() >= 'v1.1' && (
+                    <ItemArrayField
+                        label="API Versions"
+                        source="api.versions"
+                    />
+                )}
+                {controllerProps.record && QueryVersion() >= 'v1.1' && (
+                    <ArrayField label=" API Endpoints" source="api.endpoints">
+                        <Datagrid>
+                            <TextField source="host" />
+                            <TextField source="port" />
+                            <TextField source="protocol" />
+                            <FunctionField
+                                render={record => (
+                                    <a
+                                        href={buildLink(record)}
+                                        target="_blank"
+                                        rel="noopener noreferrer"
+                                    >
+                                        <LinkIcon
+                                            style={{
+                                                color: '#2196f3',
+                                                preserveAspectRatio: 'xMidYMin',
+                                            }}
+                                        />
+                                    </a>
+                                )}
+                            />
+                        </Datagrid>
+                    </ArrayField>
+                )}
+                {controllerProps.record && QueryVersion() >= 'v1.1' && (
+                    <ArrayField source="clocks">
+                        <Datagrid>
+                            <TextField source="name" />
+                            <TextField label="Ref Type" source="ref_type" />
+                        </Datagrid>
+                    </ArrayField>
+                )}
+                <ArrayField source="services">
+                    <Datagrid>
+                        <UrlField source="href" label="Address" />
+                        <TextField source="type" />
+                    </Datagrid>
+                </ArrayField>
+                {controllerProps.record && QueryVersion() >= 'v1.2' && (
+                    <ArrayField source="interfaces">
+                        <Datagrid>
+                            <TextField source="chassis_id" label="Chassis ID" />
+                            <TextField source="name" />
+                            <TextField source="port_id" label="Port ID" />
+                        </Datagrid>
```

Comment:
(Still need the separate PR to add `attached_network_device.chassis_id` and `.port_id` if `>= 'v1.3'`. Think we decided the full name was too long so we could show "Attached Chassis ID" and "Attached Port ID" - or maybe "Remote Chassis ID" and "Remote Port ID" since LLDP uses the terms local and remote.)

Reply:
Done